service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -50,10 +185,16 @@ default void bindServiceInfo(ServiceInfo serviceInfo) {
         // Bind the resolved services
         binder.bind(ResolvedServices.class).toProvider(new ResolvedServicesProvider(allServiceBindings));
 
+        // use empty list of routers
+        // no support for additional routers because the purpose of a `bindServiceInfo` is to
+        // provide the means to create a consume-only Lagom service
+        binder.bind(new TypeLiteral<List<Router>>(){}).toInstance(Collections.emptyList());
```

Comment:
Not all service bindings method should support additional routers. This one can be revisited though.